nevergrad/optimization/oneshot.py

```diff
@@ -156,7 +162,8 @@ class RandomSearchMaker(base.ConfiguredOptimizer):
     scale: float or "random"
         scalar for multiplying the suggested point values, or string:
          - "random": uses a randomized pattern for the scale.
-         - "auto": scales in function of dimension and budget (see XXX)
+         - "auto": scales in function of dimension and budget (version 1)
+         - "autolog": scales in function of dimension and budget (version 2)         
```

Comment:
'auto' and 'autolog' are not clear distinguishable names regarding both formulas